docs/changelog.md

```diff
@@ -72,6 +72,9 @@ permalink: /changelog/
 * **feature-pwa**
   * Adds the ability to create a basic shortcut with a custom label
 
+* **concept-sync**
+  * ⚠️ **This is a breaking change**: `action` param of `AuthType.OtherExternal` is now optional. Missing `action` indicates that we really don't know what external authType we've hit.
```

Comment:
nit: I don't think it's a breaking change since we're going from required -> optional.

Reply:
It's a breaking change for consumers of `AuthType`. If they relied on `action` as being always present, now it's not the case.